src/server.js

```diff
@@ -202,6 +209,53 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`,
+  getEvent,
+  async (request, response) => {
+    const { event } = request;
+    const users = event.users || {};
+    const userData = Object.values(users);
+
+    if (userData.length > 0) {
+      // Currently accessing the latitude and longitude of the first user (MVP).
+      // TODO (Chisom): Test average geolocation with multiple user locations.
+      const lat = userData[0].lat.toString();
+      const long = userData[0].long.toString();
+      const radiusMeters = "50000";
+      const type = "restaurant";
+      const minprice = "0";
+      const maxprice = "4";
+
+      const placesApiResponse = await (
+        await fetch(
+          `https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=${lat},${long}&radius=${radiusMeters}&type=${type}&minprice=${minprice}&maxprice=${maxprice}&key=${env.API_KEY_PLACES}`
+        )
+      ).json();
+      const placesApiResponseStatus = placesApiResponse.status;
+      if (placesApiResponseStatus !== "OK") {
+        console.error("Places API error Response: " + placesApiResponseStatus);
+        response.status(500).json({
+          status: 500,
+          error: { type: ERROR_BAD_PLACES_API_INTERACTION },
```

Comment:
@aivey65  I made the change. But, I believe response.status already sends information about bad API interaction to the client. That also handles the case.